src/test/unit_tests/test_bucketspace.js

```diff
@@ -223,8 +224,8 @@ mocha.describe('bucket operations - namespace_fs', function() {
             {
                 nsfs_account_config: {},
                 default_resource,
-                should_fail: !process.env.NC_CORETEST,
-                error_code: 'FORBIDDEN'
+                should_fail: true,
+                error_code: is_NC_CORETEST ? ManageCLIError.MissingUpdateProperty.code : 'FORBIDDEN'
```

Comment:
It failed in the CI.
Before this change, we could use account update without a property to update, so `should_fail: !process.env.NC_CORETEST` was `false`, and it is `true`, I added the ternary just because I didn't want to fail the containerized environment.
If you have other suggestions I would like to hear them.

[Here](https://github.com/noobaa/noobaa-core/actions/runs/8321993147/job/22769076798) you can see for example:
2024-03-18T06:30:32.7240202Z   1) bucket operations - namespace_fs
2024-03-18T06:30:32.7240782Z        update account:
2024-03-18T06:30:32.7242099Z      AssertionError [ERR_ASSERTION]: update_account_nsfs_config failed RPC ERROR FORBIDDEN Invalid nsfs_account_config,
2024-03-18T06:30:32.7243131Z       at update_account_nsfs_config (src/test/unit_tests/test_bucketspace.js:1735:16)
2024-03-18T06:30:32.7243974Z       at async Context.<anonymous> (src/test/unit_tests/test_bucketspace.js:233:13)

Reply:
if `should_fail` is failse, why code had `error-code` FORBIDDEN?

Reply:
You're asking about the older version of the code which I didn't write, but from what I understand it uses the function `update_account_nsfs_config`, and the `error_code` is used only if `should_fail` is true. Therefore, when `should_fail` is false it didn't use the `error_code` FORBIDDEN and was only used in the containerized (env we don't pass the env `NC_CORETEST` so `should_fail` is true).

Reply:
Just wanted to understand the flow, If you think it will not break anything you can ignore that